src/Client.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Sportlobster\Digits;
+
+use GuzzleHttp\Client as HttpClient;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7\Request;
+use Sportlobster\Digits\Exception\AuthenticationException;
+use Sportlobster\Digits\Exception\InvalidHostException;
+use Sportlobster\Digits\Exception\InvalidSchemeException;
+use Sportlobster\Digits\Exception\KeyMismatchException;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
+use Symfony\Component\Serializer\Normalizer\GetSetMethodNormalizer;
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class Client
+{
+    const HOST_DIGITS = 'api.digits.com';
```

Comment:
but you can't do that from the api ... so you have no way of doing a black box test from the api...